### PR TITLE
8.2XS82ECU1017

### PR DESCRIPTION
**Pull request by gduperrey**

### CODE REVIEW

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Target of the PR must be changed to 8.2.

**Comment by stormi** [comment]
/!\ Don't merge to master after merging to 8.2. We'll likely get the fixes from 8.3 SRPMs. If in doubt, we could create a card to remind us to check it does happen in the future, but at this stage of the development of 8.3 it's rather safe to assume it will be propagated automatically next time we rebase on a new SRPM from 8.3.

**Review by benjamreis — APPROVED** [review]

**Pull request merged**
